### ISSUE TITLE
No more ZCSLicense

### ISSUE THREAD

**Issue by lepoldan** [Zimbra/zinstaller#4]
Since the last scheduled downtime of license server (2023-11-06) the command:

`wget -q --no-check-certificate --no-proxy -O /tmp/zcs/ZCSLicense.xml "https://license.zimbra.com/zimbraLicensePortal/public/STLicense?IssuedToName=MyCompany&IssuedToEmail=noone@$DOMAIN"
`
return the error :

ERROR 404: (no description)

Can you do something ?

**Comment by ghost** [comment]
Yes new license server does not support getting a trial license, looking for a work-around ATM

**Comment by sollevix67** [comment]
Tried today and works fine
